src/pages/Community/PostDetail/ReplyComments.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import "./ReplyComments.scss";
+import { BsHeart } from "react-icons/bs";
+
+class ReplyComments extends Component {
+  render() {
+    const { userId, comment } = this.props;
+    return (
+      <div className="replyCommentComponentWrap">
+        <li className="replyCommentComponent">
+          <div className="replyCommentUser">
+            <img
+              className="replyCommentUserPicture"
+              src="/images/Communityimages/chair.jpg"
+              alt=""
+            />
```

Comment:
```suggestion
class ReplyComments extends Component {
  render() {
    const { userId, comment } = this.props;
    return (
      <div className="ReplyComments">
        <li className="comment">
          <div className="replyUser">
            <img
              className="picture"
              src="/images/Communityimages/chair.jpg"
              alt="picture"
            />
```
- 컴포넌트 최상위 클래스명은 클래스명과 동일하게 지어주세요. 이 이름을 기준으로 컴포넌트 내 셀렉터들 전체를 감싸주시면 됩니다.
- nesting이 잘 되었다면 이하 속성들은 굳이 반복되는 이름을 부여할 필요 없습니다. 간결하게 작성해주세요
- alt 속성 채워주세요!

Reply:
수정했습니다.